### PR TITLE
0.5 alpha compressed commits

### PR DESCRIPTION
**Pull request by esheppa**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -425,13 +417,7 @@
 #![cfg_attr(feature = "rustc-serialize", allow(deprecated))]
 #![cfg_attr(docsrs, feature(doc_cfg))]
 
-#[cfg(feature = "oldtime")]
-#[cfg_attr(docsrs, doc(cfg(feature = "oldtime")))]
-extern crate time as oldtime;
-#[cfg(not(feature = "oldtime"))]
-mod oldtime;
 // this reexport is to aid the transition and should not be in the prelude!
```
Comment (djc) [inline src/lib.rs:414]: Should remove this line, too, right?

File: src/lib.rs [hunk 2]
```diff
@@ -425,13 +417,7 @@
 #![cfg_attr(feature = "rustc-serialize", allow(deprecated))]
 #![cfg_attr(docsrs, feature(doc_cfg))]
 
-#[cfg(feature = "oldtime")]
-#[cfg_attr(docsrs, doc(cfg(feature = "oldtime")))]
-extern crate time as oldtime;
-#[cfg(not(feature = "oldtime"))]
-mod oldtime;
```
Comment (djc) [inline src/lib.rs:432]: Without this module declaration (and the import below, I suppose), I'm pretty sure this commit does not compile (and test). Would be nice to fix that properly.

Comment (esheppa) [reply src/lib.rs:432]: ah yikes I must have grabbed this in the wrong commit

Comment (esheppa) [reply src/lib.rs:432]: Hmm looks like I wiped out some commits when redoing them, will separate them again (I'm learning a bunch of `git` features I never knew existed!)

Comment (djc) [reply src/lib.rs:432]: If you did this using rebase -i, watch out for amending commits vs continuing the rebase, that's usually how I accidentally squash commits.

Comment (esheppa) [reply src/lib.rs:432]: Classic, that is exactly what I did! Should be right now, I've checked that it can compile at each commit

**Review by djc — APPROVED** [review]
Awesome! Just needs a rebase.

**Comment by djc** [comment]
I've pushed a 0.4.x branch from the HEAD of main before merging this.

**Pull request merged**